esp32c6-hal/.cargo/config.toml

```diff
@@ -0,0 +1,27 @@
+[target.riscv32imac-unknown-none-elf]
+runner = "espflash flash --monitor"
+rustflags = [
+  "-C", "link-arg=-Tlinkall.x",
+
+  # comment the cfgs below if you do _not_ wish to emulate atomics.
```

Comment:
we don't need all of these overrides since `imac` already enables them

Reply:
Thanks, just realized I had copy-pasted this and not updated it last night 😅  Fixed now!